sqlglot/dialects/redshift.py

```diff
@@ -124,27 +126,6 @@ def _parse_approximate_count(self) -> t.Optional[exp.ApproxDistinct]:
             self._retreat(index)
             return None
 
-        def _parse_query_modifiers(
```

Comment:
why was this one removed?

Reply:
it was moved to the base class and altered slightly - now controlled with a class constant because BQ's logic is similar